api.yaml

```diff
@@ -14,284 +14,80 @@ externalDocs:
           schema:
-            type: object
-            properties:
-              application_id:
-                type: string
-                description: Unique identifier of the application
-                example: kio
-              versions:
-                type: array
-                items:
-                   type: object
-                   properties:
-                     version_id:
-                       type: string
-                       description: Unique identifier of the version
-                       example: somestring
-                     data:
-                       type: array
-                       items:
-                         type: object
-                         properties:
-                           change_date:
-                             type: string
-                             format: date-time
-                             description: Point in time when the instance count of the version changed.
-                             example: '2015-04-25T16:25:00.000Z'
-                           instance_count:
-                             type: integer
-                             format: int32
-                             description: amount of instances at the gi
-                             example: 4
+            type: array
+            items:
+              type: object
+              properties:
+                version_id:
+                  type: string
+                  description: some version id
+                metadata:
+                  type: object
+                  properties:
+                    scm_source:
+                      type: object
+                      properties:
+                        repo_url:
+                          type: string
+                        git_hash:
+                          type: string
+                        commiter:
+                          type: string
+                    docker_source:
+                      type: string
+                    pull_request:
+                      type: object
+                      properties:
+                        link:
+                          type: string
+                events:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      timestamp:
+                        $ref: '#/definitions/Timestamp'
+                      count:
+                        type: integer
+                        format: int32
+                        description: amount of instances at the given change_date
+                        example: 4
+                      instances:
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            event_type:
+                              type: string
+                              enum: [ 'scaledUp', 'scaledDown']
+                            instance_id:
```

Comment:
i think it should be more generic. For example on GCP there are no accounts but "projects". 
``` yaml
Event:
    type: object
    properties:
      timestamp:
        $ref: '#/definitions/Timestamp'
      count:
        type: integer
        format: int32
        description: amount of instances at the given change_date
        example: 4
      entities:
        type: array
        items:
          $ref: "#/definitions/Entity"
          
  Entity:
    type: object
```
So if the attributes of some entity/instance are not used by the UI, than displaying them, then maybe we should leave it as an empty object which could contain individual attributes.